src/main/java/org/folio/innreach/domain/listener/KafkaCirculationEventListener.java

```diff
@@ -46,6 +46,7 @@ public void handleLoanEvents(List<ConsumerRecord<String, DomainEvent<LoanDTO>>>
           break;
         case UPDATED:
           transactionActionService.handleLoanUpdate(event.getData().getNewEntity());
+          transactionActionService.borrowerRenewLoan(event.getData().getNewEntity());
```

Comment:
merge your logic into `handleLoanUpdate`, add `break;` at the end of this case

Reply:
Done